File: typescript/30-count-in-range.ts
// Contar elementos en rango — Fácil · Arrays
// Cuenta los elementos dentro del rango inclusivo [min, max].
// Fuente: https://coding-challenges.dev/problems/typescript-contar-elementos-en-rango

export function contarEnRango(numbers: number[], min: number, max: number): number {
  return numbers.filter((number) => min <= number && number <= max).length;
}

File: python/30_count_in_range.py
"""Contar elementos en rango — Fácil · Arrays

Cuenta los elementos dentro del rango inclusivo [min, max].

Fuente: https://coding-challenges.dev/problems/python-contar-elementos-en-rango
"""


def contar_en_rango(numbers: list[int], min: int, max: int) -> int:
    return sum(min <= number <= max for number in numbers)
